src/lib/dnssd/Advertiser.h

```diff
@@ -101,6 +101,13 @@ class BaseAdvertisingParams
     }
     Optional<bool> GetTcpSupported() const { return mTcpSupported; }
 
+    Derived & SetICDOperatingAsLIT(Optional<bool> operatesAsLIT)
+    {
+        mICDOperatesAsLIT = operatesAsLIT;
+        return *reinterpret_cast<Derived *>(this);
+    }
+    Optional<bool> GetICDOperatingAsLIT() const { return mICDOperatesAsLIT; }
```

Comment:
Just to use the same terminalogy every where, this should be renamed.

Reply:
I don't really agree. it is a separate structure and this variable used internally by the getter setter of the same name. In this case, reducing the variable name seems more important than matching another variable in another class